openrl/envs/common/registration.py

```diff
@@ -155,6 +155,18 @@ def make(
             env_fns = make_PettingZoo_envs(
                 id=id, env_num=env_num, render_mode=convert_render_mode, **kwargs
             )
+        elif (
+            "envpool:" in id
```

Comment:
Envpool can not be installed on many platforms (such as Windows), so you should use the envpool as an external env which is out of OpenRL. Moreover, with the envpool as an external env, you don't need to write test code for codecov, because we only track the code in the openrl folder.

Examples:
- https://openrl-docs.readthedocs.io/en/latest/custom_env/index.html#moreexamples1
- https://github.com/OpenRL-Lab/openrl/tree/main/examples/isaac
- https://github.com/OpenRL-Lab/openrl/tree/main/examples/smac